docs/about/contributing.md

```diff
@@ -7,21 +7,139 @@ nav_order: 2
 
 # Contributing to the nih-cfde Use Case Repository
 
-Hello, and thank you for wanting to contribute to the CFDE Use Case
-Repository\!
+Hello, and thank you for your interest in contributing to the CFDE Use Case Repository!
 
 By contributing to this repository, you agree:
 
 1.  To obey the [Code of Conduct](./CODEOFCONDUCT.md)
-2.  To release all your contributions under the same terms as the
-    license itself: the [Creative Commons Zero](./LICENSE.md) (aka
-    Public Domain) license
+2.  To release all your contributions under the same terms as the license itself: the [Creative Commons Zero](./LICENSE.md) (aka Public Domain) license
 
-If you are OK with these two conditions, then we welcome both you and
-your contribution\!
+If you are OK with these two conditions, then we welcome both you and your contribution!
 
-If you have any questions about contributing, please [open an
-issue](https://github.com/nih-cfde/usecases/issues/new) and we
-will lend a hand ASAP.
+## How to add content
 
-Thank you for being here and for being a part of the CFDE project.
+If you have a suggestion for a new use case concept and are not familiar with GitHub, you can [email your idea to us](mailto:autohelp+int+851+6545985337373134556@CFDE.groups.io).
+
+If you are familiar with GitHub please either:
+
+  - [open an issue](https://github.com/nih-cfde/usecases/issues/new) describing your idea
+  
+  - or write up your use case and submit it as a pull request (PR) at [https://github.com/nih-cfde/usecases](https://github.com/nih-cfde/usecases). Please follow the [Use Case Style Guide](#usecasestyle) below.
+
+### PR format
+If you are submitting a pull request:
```

Comment:
Let's make an actual PR template and have the instructions here tell them to fill it in completely

Reply:
i drafted the steps for PR to preview and then PR to master... not sure if this is what we want ppl to do, please edit!